### ISSUE TITLE
Building causes an error with a newer version of Java

### ISSUE THREAD

**Issue by hchittanuru3** [phenopackets/phenopacket-schema#262]
When trying to build the phenopacket-schema using maven, I get this error:
```
Execution attach-javadocs of goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.0:jar failed: Cannot invoke "org.apache.commons.lang3.JavaVersion.atLeast(org.apache.commons.lang3.JavaVersion)" because "org.apache.commons.lang3.SystemUtils.JAVA_SPECIFICATION_VERSION_AS_ENUM" is null
```

Was wondering if anyone had any advice on how to fix this?

**Comment by julesjacobsen** [comment]
Which newer version of Java?

**Comment by hchittanuru3** [comment]
15.0.1

**Comment by julesjacobsen** [comment]
Hi @hchittanuru3, thanks for the feedback - I've pushed a fix to master and v1.1. Please reopen if this doesn't work for you.
